### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Name property is empty for "Run on web app for containers dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1401]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish to web app for containers
4. Observe the name property for the dialog.

 **Actual Result:**
Name property is empty for the Dialog.

**Narrator Behaviour:**
Docker File
editable text
button..

**NVDA:**
dialog  docker file
image and tag
Run on web app for conatiners
pane

**Expected Result:**
Name property should be "Run on web app for containers" dialog

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app fro containers-dialog-inspect](https://user-images.githubusercontent.com/22956130/35684657-9aaac724-078d-11e8-856b-25d4787f674f.jpg)
![run on web app fro containers-dialog-narrator](https://user-images.githubusercontent.com/22956130/35684659-9dcafe10-078d-11e8-947f-041b80735812.jpg)
![run on web app fro containers-dialog-nvda](https://user-images.githubusercontent.com/22956130/35684661-9feba780-078d-11e8-8219-68f83df524cd.jpg)

**Comment by Eskibear** [comment]
We just cannot set title bar content for `org.eclipse.jface.dialogs.TitleAreaDialog`. Eclipse doesn't expose the related UI API. `title` of the dialog is already set to "Push Image", which is showing in the title area.

Ref : #1407

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by jdneo** [comment]
Can show the name property on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0

![image](https://user-images.githubusercontent.com/6193897/47149795-3ad3f000-d307-11e8-9bee-4a1c32bafb30.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749836
